packages/core/src/forms/control-inline/control-inline.element.ts

```diff
@@ -99,7 +99,7 @@ export class CdsInternalControlInline extends CdsControl {
     );
   }
 
-  updated(props: Map<string, any>) {
+  updated(props: PropertyValues<any>) {
```

Comment:
Why are the others `PropertyValues<this>` but this one is `PropertyValues<any>`?

Reply:
https://github.com/vmware/clarity/pull/6667/files#r802807628